src/Abp.Zero.EntityFramework/EntityHistory/EntitySnapshotManager.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+using Abp.Dependency;
+using Abp.Domain.Entities;
+using Abp.Domain.Repositories;
+using Abp.EntityFramework;
+using Abp.EntityFramework.Repositories;
+using Abp.Events.Bus.Entities;
+using Abp.Json;
+
+namespace Abp.EntityHistory
+{
+    public class EntitySnapshotManager : EntitySnapshotManagerBase
+    {
+        private readonly IRepository<EntityChange, long> _entityChangeRepository;
```

Comment:
maybe we can move this to `EntitySnapshotManagerBase` since `EntityChange` is available in Abp core package